src/modules/competition/infrastructure/api/v1/competition_routes.py

```
@@ -0,0 +1,1135 @@
+
+
+@router.get(
+    "",
+    response_model=List[CompetitionResponseDTO],
+    status_code=status.HTTP_200_OK,
+    summary="Listar competiciones con filtros",
+    description="Obtiene lista de competiciones con filtros opcionales.",
+    tags=["Competitions"],
+)
+async def list_competitions(
+    current_user: UserResponseDTO = Depends(get_current_user),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)